ui/src/components/Publication/Creation/DataStatements.tsx

```diff
@@ -4,15 +4,13 @@ import * as OutlineIcons from '@heroicons/react/outline';
 
 import * as Components from '@components';
 import * as Stores from '@stores';
+import * as Config from '@config';
 
 const ethicalStatementOptions: string[] = [
     'The results in this publication involved human or animal subjects.',
     'The results in this publication do <strong>not</strong> involve human or animal subjects.'
 ];
-const dataPermissionsOptions: string[] = [
-    'The results in this publication involved access to owned or copyrighted materials.',
-    'The results in this publication do <strong>not</strong> involve access to materials owned or copyrighted materials (except those in the private ownership of the authors).'
```

Comment:
It might be better to store this as a keyed object, as if we want to change the text in the future, previous data will be stored differently on the database but will mean the same thing :)